docsite/rst/intro_configuration.rst

```diff
@@ -70,7 +70,7 @@ Actions are pieces of code in ansible that enable things like module execution,
 This is a developer-centric feature that allows low-level extensions around Ansible to be loaded from
 different locations::
 
-   action_plugins = ~/.ansible/plugins/action_plugins/:/usr/share/ansible_plugins/action_plugins
+   action_plugins = ~/.ansible/plugins/action:/usr/share/ansible/plugins/action
```

Comment:
I'll remove it then but out of curiosity, @sivel why do you follow a different convention in `constants.py`?